test/integration/test_create.py

```diff
@@ -394,3 +395,23 @@ def test_v3_no_workdir(cli, build_dir_and_ee_yml):
 
     assert "WORKDIR" not in text
     assert "mkdir -p /runner" not in text
+
+
+def test_v3_set_user_id(cli, build_dir_and_ee_yml):
+    """
+    Test that a custom 'options.user' sets it
+    """
+    tmpdir, eeyml = build_dir_and_ee_yml(
+        """
+        version: 3
+        options:
+          user: bob
+        """
+    )
+    cli(f'ansible-builder create -c {tmpdir} -f {eeyml} --output-filename Containerfile')
+
+    containerfile = tmpdir / "Containerfile"
+    assert containerfile.exists()
+    text = containerfile.read_text()
+
+    assert "USER bob" not in text
```

Comment:
I assume this should be:
```suggestion
    assert "USER bob" in text
```